src/modules/template-details/TemplateDetails.tsx

```diff
@@ -56,105 +59,129 @@ const TemplateDetails = ({ match }: TemplateDetails.Props) => {
     setConfirmDeleteOpen(false);
   }, []);
 
+  const forkTemplateHandler = async () => {
+    info.incrementForks();
+    setForkInProgress(true);
+    forkTemplate(match.params.id).then(
```

Comment:
Use async / await instead of this promise based approach

Reply:
done